ci/bamboo/test-linux.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# -----------------------------------------------------------------------------
+
+# Run NuPIC tests on Linux.
+
+# ASSUMES:
+#   1. Current working directory is root of nupic source tree
+#   2. The nupic wheel is in the current working directory
+#
+# OUTPUTS:
+#
+# test results: nupic test results will be located in the subdirectory
+#               test_results of the source tree's root directory with the
+#               the following content:
+#
+#               nupic-test-results-unit.xml
+#               nupic-test-results-integration.xml
+
+
+set -o errexit
+set -o xtrace
+
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+NUPIC_ROOT_DIR="$( cd "${MY_DIR}/../.." && pwd )"
+
+TEST_RESULTS_DIR="${NUPIC_ROOT_DIR}/test_results"
+
+
+# ZZZ Temp workaround for linux until we have proper manylinux nupic.bindings
+# wheels on PyPi
+while read line; do
+    if [[ $line == nupic.bindings* ]];
+    then
+        IFS='=' read -ra ADDR <<< "$line"
+        NUPIC_BINDINGS_VERSION="${ADDR[2]}"
```

Comment:
I *think* this line is trying to figure out which nupic.bindings version we got but we should really be specifying that / know it ahead of time.